app/controllers/shipit/shipit_controller.rb

```diff
@@ -32,9 +32,9 @@ def ensure_required_settings
 
     def force_github_authentication
       if current_user.logged_in?
-        team = Shipit.github_team
-        if team && !current_user.in?(team.members)
-          render text: "You must be a member of #{team.handle} to access this application.", status: :forbidden
+        teams = Shipit.github_teams
+        if teams && teams.none? { |t| current_user.in?(t.members) }
```

Comment:
Fine. I'll add the missing association to `User`, but I don't think we should preload teams for users, though.